docs/file-reference.md

```diff
@@ -1067,6 +1067,16 @@ NGINX_VERSION=1.13 kedge apply -f nginx.yaml
 
 You can use variables anywhere in the Kedge file. Variable names are enclosed in double square brackets (`[[ variable_name ]]`). For example `[[ IMAGE_NAME ]]` will be replaced with value of environment variable `$IMAGE_NAME`.
 
+You can specify default value of variable if it's not set. for example, `[[ TAG:latest ]]`
```

Comment:
You need to move these two sentences BELOW the example (after the yaml code) or else the formatting is a bit messed up.

Reply:
`You can specify a default value of a variable if it is not set.`

Reply:
`For example: `[[ TAG:latest ]]`